api/resources/verify.py

```diff
@@ -0,0 +1,17 @@
+import requests
```

Comment:
Should this script locate in `api/utils` ?

Reply:
Fix in f71aa2f.